dataland-inbound-proxy/custom_errors.html

```diff
@@ -0,0 +1,46 @@
+                <path d="M53.405 4.66854V23.7735H48.2434V4.66854H41.8254V0.0874329H59.823V4.66854H53.405Z" fill="#E67F3F"/>
+                <path d="M75.7491 23.7735L74.0512 18.1404H66.139L64.4412 23.7735H59.2117L67.0219 0.0874329H73.406L81.1144 23.7735H75.7491ZM70.146 4.77034H69.9762L67.3615 13.7629H72.7947L70.146 4.77034Z" fill="#E67F3F"/>
+                <path d="M84.4084 23.7735V0.0874329H89.57V19.1924H98.6028V23.7735H84.4084Z" fill="#E67F3F"/>
+                <path d="M117.585 23.7735L115.887 18.1404H107.975L106.277 23.7735H101.048L108.858 0.0874329H115.242L122.95 23.7735H117.585ZM111.982 4.77034H111.812L109.197 13.7629H114.631L111.982 4.77034Z" fill="#E67F3F"/>
+                <path d="M133.477 12.5752L131.202 7.82442H131.1V23.7735H126.244V0.0874329H131.881L138.367 11.2857L140.642 16.0365H140.744V0.0874329H145.6V23.7735H139.963L133.477 12.5752Z" fill="#E67F3F"/>
+                <path d="M150.83 0.0874329H159.76C161.323 0.0874329 162.749 0.336283 164.039 0.833984C165.33 1.33168 166.427 2.07258 167.333 3.05667C168.239 4.04076 168.94 5.2737 169.438 6.75549C169.936 8.23728 170.185 9.96227 170.185 11.9304C170.185 13.8986 169.936 15.6236 169.438 17.1054C168.94 18.5872 168.239 19.8201 167.333 20.8042C166.427 21.7883 165.33 22.5292 164.039 23.0269C162.749 23.5246 161.323 23.7735 159.76 23.7735H150.83V0.0874329ZM159.76 19.1924C161.3 19.1924 162.511 18.7625 163.394 17.9029C164.277 17.0432 164.718 15.6632 164.718 13.7629V10.098C164.718 8.19769 164.277 6.8177 163.394 5.95804C162.511 5.09837 161.3 4.66854 159.76 4.66854H155.991V19.1924H159.76Z" fill="#E67F3F"/>
+                <path fill-rule="evenodd" clip-rule="evenodd" d="M19.1241 7.37507C18.6136 5.84047 17.8943 4.56355 16.966 3.5443C16.0378 2.52515 14.9123 1.75785 13.5896 1.24239C12.2669 0.726943 10.805 0.469269 9.20377 0.469269H0.0493164V5.37541H4.95886H5.56989H8.86276C10.4571 5.37541 12.4918 5.65884 13.4062 6.5492C14.3206 7.43946 14.7779 8.86868 14.7779 10.8368V14.6324C14.7779 16.6005 14.3206 18.0297 13.4062 18.9199C12.4918 19.8103 10.4571 20.0938 8.86276 20.0938H4.95886V10.2816H0.0493164V25H9.20377C10.805 25 12.2669 24.7422 13.5896 24.2267C14.9123 23.7113 16.0378 22.944 16.966 21.9248C17.8943 20.9056 18.6136 19.6287 19.1241 18.0941C19.6346 16.5595 19.6875 14.7729 19.6875 12.7346C19.6875 10.6962 19.6346 8.90967 19.1241 7.37507Z" fill="#E67F3F"/>
+            </g>
+            <defs>
+                <clipPath id="clip0_8_619">
+                    <rect width="170" height="25" fill="white"/>
+                </clipPath>
+            </defs>
+        </svg>
+    </div>
+</div>
+<div style="background-color:#FFFFFF;" class="alignment">
+    <h2>Sorry an error occurred!</h2>
+    <p>Whoops! Something went wrong!</p>
+    <p>Go back to <a href="/" class="text-primary" title="back to landing page">Dataland</a></p>
+</div>
+</body>
+<style scoped>
+    .bg-black{
```

Comment:
\<style scoped\> sollte nur innerhalb von Vue componenten verwendet werden.  Im "puren" HTML ist das feature depricated und sollte nicht verwendet werden (ref https://www.w3schools.com/tags/att_scoped.asp#:~:text=The%20scoped%20attribute%20is%20a,deprecated%20and%20will%20be%20removed.)

Reply:
scoped entfernt und was neues gelernt :)